unit-tests/src/test/scala/java/lang/DoubleSuite.scala

```diff
@@ -194,6 +194,7 @@ object DoubleSuite extends tests.Suite {
     assert(Double.parseDouble("Infinity") == Double.POSITIVE_INFINITY)
     assert(Double.parseDouble("-Infinity") == Double.NEGATIVE_INFINITY)
     assert(Double.isNaN(Double.parseDouble("NaN")))
+    assertThrows[NumberFormatException](Double.parseDouble("potato"))
```

Comment:
Needs another test case to check that `Double.parseDouble("0.0potato")` throws as well.